mappings/net/minecraft/entity/EntityType.mapping

```diff
@@ -95,7 +95,7 @@ CLASS net/minecraft/class_1299 net/minecraft/entity/EntityType
 		ARG 0 type
 	METHOD method_5891 getSpawnGroup ()Lnet/minecraft/class_1311;
 	METHOD method_5892 getEntityFromNbt (Lnet/minecraft/class_2487;Lnet/minecraft/class_1937;)Ljava/util/Optional;
```

Comment:
Hmm, this might need name optimization, need to view actual code for a better name